### PR TITLE
documentation/1319

### LINKED ISSUE #1319: Lint and update docs markdown files
Our airport guide files need to be updated so they are properly formatted.

Steps:
1. install markdownlint (https://github.com/DavidAnson/vscode-markdownlint)
1. address lint errors in each markdown file within `documentation/airport-guides`

Effort should be made to not only address linting issues, but also better structure content.
screen shot 2019-02-06 at 11 17 35 pm

Example like the above are written as a collection of links broken by a new line.  We could use an unordered list `*` here which would give us more control over the presentation within the UI.

### PR DESCRIPTION
**Pull request by n8rzz**
Resolves #1319 

addresses lint errors throughout all *.md files in the documentation/ directory

### CODE REVIEW

File: documentation/development-processes-checklists.md [hunk 1]
```diff
@@ -69,12 +75,13 @@ At least three days prior to the end of the sprint, we will create a `release` b
 1. On Heroku, change staging app to point to this new release branch.
 1. Broadcast publishing of testing app and seek feedback and/or bug reports. Any bugs should be reported to the `#bugs` room in slack and will be triaged from there.
 1. Merge any applicable bugfix branches into `release/#.#.#`.
-    - Include summary of `Merge bugfix/### (#[PRNumber])`
+    - Include summary of `Merge bugfix/ (#[PRNumber])`
```
Comment (erikquinn) [inline documentation/development-processes-checklists.md]: I smell a find-and-replace whoopsie!!! 😃

Comment (n8rzz) [reply documentation/development-processes-checklists.md]: `cmd + d` did it's job, but you are correct ☹️

File: documentation/aircraft-format.md [hunk 2]
```diff
@@ -1,53 +1,53 @@
...
-Recommended sources:      
-¹ https://www.icao.int/publications/DOC8643/Pages/Search.aspx   
-² https://www.faa.gov/documentLibrary/media/Order/2017-03-07_FAA_Order_JO_7360.1B_Aircraft_Type_Designators.pdf   
-³ https://contentzone.eurocontrol.int/aircraftperformance/default.aspx    
-⁴ https://www.easa.europa.eu/document-library/type-certificates or    
-http://www1.airweb.faa.gov/Regulatory_and_Guidance_Library/rgMakeModel.nsf/MainFrame?OpenFrameSet   
-(Try to find the correct .pdf file and search for "VMO" or "MMO")    
+Recommended sources:
+
+1. [https://www.icao.int/publications/DOC8643/Pages/Search.aspx](https://www.icao.int/publications/DOC8643/Pages/Search.aspx)
```
Comment (Marcel510) [inline documentation/aircraft-format.md]: I'd personally keep the superscript numbers here as this makes more clear that those are footnotes that refer to the text.

Comment (n8rzz) [reply documentation/aircraft-format.md]: the `1.` at the bottom is an ordered list, so they will still match up with the superscript references above

**Pull request merged**